core/src/main/java/bisq/core/trade/TradeManager.java

```diff
@@ -297,6 +297,15 @@ private void initPendingTrades() {
                                 trade.getId());
                         tradesWithoutDepositTx.add(trade);
                     }
+
+            try {
+                DonationAddressValidation.validate(trade.getDelayedPayoutTx(),
```

Comment:
This causes problems at startup. I think it's because it's throwing a runtimeexception due to `        checkNotNull(delayedPayoutTx, "delayedPayoutTx must not be null");` in `DonationAddressValidation.validate`